src/deployment/set_admins.py

```diff
@@ -5,12 +5,20 @@
 
 import argparse
 import json
+from time import sleep
 from typing import List, Optional
 from uuid import UUID
 
 from azure.common.client_factory import get_client_from_cli_profile
 from azure.cosmosdb.table.tableservice import TableService
 from azure.mgmt.storage import StorageManagementClient
+import logging
+
+## Disable logging from storageclient. This modules display and error message
+## when a resource is not found even if the exception is raised and handled internally.
+## This happen when a table does not exist. An error message is displayed but the exception is
+## handled by the library.
+logging.getLogger('azure.cosmosdb.table.common.storageclient').disabled = True
```

Comment:
We don't control this name, so we should ensure that `getLogger` returns an object. Otherwise the `.disabled` attr lookup could fail and throw an exception at module load time.